app/models/shipping_office.rb

```diff
@@ -0,0 +1,3 @@
+class ShippingOffice < Office
+  has_many :transportation_offices, class_name: 'Office', foreign_key: :shipping_office_id, dependent: :destroy
+end
```

Comment:
Using [single table inheritance](http://guides.rubyonrails.org/association_basics.html#single-table-inheritance) here since the two types of offices share essentially the same data structure.